projects/plugins/jetpack/extensions/blocks/like/like.php

```diff
@@ -62,13 +62,17 @@ function render_block( $attr, $content, $block ) {
 		return;
 	}
 
-	add_action( 'wp_footer', __NAMESPACE__ . '\render_iframe', 25 );
+	// make sure we have `jetpack_likes_master_iframe` defined
+	require_once JETPACK__PLUGIN_DIR . 'modules/likes/jetpack-likes-master-iframe.php';
```

Comment:
Is it expected that we load the `jetpack-likes-master-iframe.php` directly? Shouldn't we load this instead:

```suggestion
	require_once JETPACK__PLUGIN_DIR . 'modules/likes.php';
```

I see that this is how `render_iframe()` was doing it.

Reply:
Loading only `jetpack-likes-master-iframe.php` is more precise for the Like block. That's all we need to render the block. The `modules/likes.php` code does only things that the block doesn't need:
- setup hooks to render the Like button as part of `the_content`. That's an old-fashioned way to do it, the modern block-based approach is to simply have the Like block in the theme template. Or added through block hooks.
- setup WP Admin UI to configure the Likes module. Also not relevant when rendering the block on the frontend.